src/memefactory/server/syncer.cljs

```diff
@@ -148,7 +148,7 @@
                        :param-change/db db
                        :param-change/key key
                        :param-change/value (bn/number value)
-                       :param-change/original-value (:param/value (db/get-param key db))
+                       :param-change/original-value (:param/value (first (db/get-all-params db [key])))
```

Comment:
since you use get-all and then use `first` on the result, why not just write SQL that retrieves one row?

Reply:
Same as other comment, not worried about perf, this code is only going to execute on param change creation so very rarely, and when it does it deals with about 10 rows always. But if you think it is important we can change it OC.

Reply:
Similar reasons as above, lets keep the db operations performant and it will pay off in the future. :)

Reply:
Well this is returning one row already, changed the name to get-params to make it  more clear.